src/cleed++/Core/MillerIndex.cc

```
@@ -0,0 +1,68 @@
+using namespace cleed;
+
+MillerIndex::MillerIndex(
+  double h, 
+  double k, 
+  double l) {
+  this->h = h;
+  this->k = k;
+  this->l = l;
+}
+
+MillerIndex::~MillerIndex() {
```

Clang (14.0.6)

[warning] use '= default' to define a trivial destructor

(modernize-use-equals-default)